### PR TITLE
Add invoices.download documentation

### PR DESCRIPTION
**Pull request by jenssegers**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1943,6 +1943,22 @@ Get details for a single invoice.
...
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 307 (application/json;charset=utf-8)
+
+    + Headers
+
+              Location: The url of the downloadable file. This will remain valid for 30 seconds.
```
Comment (WouterSioen) [inline apiary.apib:1960]: Q: the indentation looks quite weird compared to the other endpoints. Is this intended?

**Review by WouterSioen — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (jenssegers) [reply apiary.apib:1960]: https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#example-18

Comment (WouterSioen) [reply apiary.apib:1960]: TIL

 :ok: by @WouterSioen on 30/05/2018 10:50:55

File: src/05-invoicing/invoices.apib [hunk 2]
```diff
@@ -161,6 +161,22 @@ Get details for a single invoice.
...
 
+### invoices.download [POST /invoices.download]
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
```
Comment (rathesDot) [inline src/05-invoicing/invoices.apib:172]: Do I understand that right, that we actually only offer `pdf` as format but leave it open to have more supported formats in the future?

**Review by rathesDot — APPROVED** [review]

File: src/05-invoicing/invoices.apib (hunk 2 above)
Comment (WouterSioen) [reply src/05-invoicing/invoices.apib:172]: Yep, this is to ensure we can add formats without breaking backwards compatibility.

**Review by steftrenson — APPROVED** [review]

**Pull request merged**